src/html/sandbox.html

```diff
@@ -118,12 +147,21 @@ <h6>Text:</h6>
                     case "renderDomains":
                         var domainSource = document.getElementById('addDomains').innerHTML;
                         var htmlDomainTemplate = Handlebars.compile(domainSource);
-                            event.source.postMessage({
+                        event.source.postMessage({
                             content: 'domain-list',
                             html: htmlDomainTemplate(event.data.context),
                             callback: event.data.callback
                         }, event.origin);
                         break;
+                    case "renderInputIDs":
```

Comment:
If we combine the above templates into one generic one, you should be able to combine these two cases too.